packages/ag-charts-community/src/chart/mapping/themes.ts

```diff
@@ -56,8 +116,6 @@ export function getChartTheme(value?: string | ChartTheme | AgChartTheme): Chart
         return lightTheme();
     }
 
-    value = value as AgChartTheme;
```

Comment:
Note: this expression is no longer necessary because the two if-statements above remove `ChartTheme | undefined | string` from the union, leaving `AgChartTheme` the only remaining type.